src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -456,16 +450,14 @@ private SynthesizedDefinitions GetCacheOfSynthesizedDefinitions(TNamedTypeSymbol
             {
                 return _synthesizedDefs.GetOrAdd(container, _ => new SynthesizedDefinitions());
             }
-            else
-            {
-                SynthesizedDefinitions defs;
-                if (!_synthesizedDefs.TryGetValue(container, out defs))
-                {
-                    defs = null;
-                }
 
-                return defs;
+            SynthesizedDefinitions defs;
+            if (!_synthesizedDefs.TryGetValue(container, out defs))
+            {
+                defs = null;
```

Comment:
This assignment is actually redundant: `TryGetValue` sets its out parameter to null if it returns false. I would like to remove the assignment and `if` completely. But I'm not sure if it would be unanimously accepted.